relate/static/js/fullcalendar.js

```diff
@@ -21,6 +22,7 @@ export function setupCalendar(domEl, events, initialDate) {
     },
     initialDate,
     events,
+    defaultTimedEventDuration: '00:00:00.001',
```

Comment:
Wouldn't this render events that specify no end time as really narrow slivers? (Could you post a screenshot of an event that makes use of this?)

Reply:
It works well after this patch in `dayGridMonth` view and `listWeek`, and as you expected, the sliver is narrow in the `timeGridWeek` view. 

**Before patched:**
![5a8cca94fddc130dd71e34ad3197d9a](https://user-images.githubusercontent.com/7379654/159461762-83dfc44e-d0d0-400a-b8d1-178543faf3b0.png)

![34718bfa01985ac0898b9b106299cd0](https://user-images.githubusercontent.com/7379654/159461791-5ad029f3-9e1b-48d1-a39d-48f323134e5a.png)

![d67304f45b5ecf13fe616923f64df76](https://user-images.githubusercontent.com/7379654/159461810-43e3936e-cfa0-4e9d-b0bd-590d3cfbdac6.png)

--------------------------------------------------------------------------------------------------------------------------------
**After patched:**

![b525113195b2dd1ca49b6d3ec4dd068](https://user-images.githubusercontent.com/7379654/159461856-d029f84f-c257-41cb-bf8e-f5bcc74bc878.png)

![58402f39e6b5f64cc2330cec59cf8e3](https://user-images.githubusercontent.com/7379654/159461892-bddaeccb-f233-4ba5-bf60-6f5b3f978d7c.png)

![e062a5fd531835d0a416b0a8615a6c0](https://user-images.githubusercontent.com/7379654/159461921-48329fed-ac08-4fb6-a5fe-afc7f47b1561.png)


I had tried to add that `defaultTimedEventDuration` option only to `dayGridMonth` view, however, it failed because the option is not functioning as a (`view-specific-option`)[https://fullcalendar.io/docs/view-specific-options]. Any suggestions?